#!/usr/bin/env python
PACKAGE = "rcbigcar"

from dynamic_reconfigure.parameter_generator_catkin import *
from math import pi

gen = ParameterGenerator()

gen.add("Kp", double_t, 0, "Motor Controller Kp",  0.005,     0,   10)
gen.add("Ki", double_t, 0, "Motor Controller Ki",  0.2,       0,   10)
gen.add("Kd", double_t, 0, "Motor Controller Kd",  0.00004,   0,   10)
gen.add("Kf", double_t, 0, "Motor Controller Kf",  0,        -1,   10)
gen.add("KmaxI", double_t, 0, "Motor Controller KmaxI",  1.0, 0,   10)

gen.add("MaxVel", double_t, 0, "Motor Max Speed (rad/s)", 167, 0, 200)

exit(gen.generate(PACKAGE, "chassis", "Chassis"))

